stackgl_modules/package.json

```diff
@@ -28,7 +28,7 @@
     "gl-spikes2d": "^1.0.2",
     "gl-spikes3d": "^1.0.11",
     "gl-streamtube3d": "^1.4.2",
-    "gl-surface3d": "^1.6.1",
+    "gl-surface3d": "github:hborchardt/gl-surface3d#6c75a5b44781c2aaec543237312df1f14274feac",
```

Comment:
Please open a PR to `gl-vis/gl-surface3d` and provide the link here.
Thank you!

Reply:
See https://github.com/gl-vis/gl-surface3d/pull/21

Reply:
`gl-surface3d@1.6.2` is published. Please install new patch.